torchgeo/models/changestar.py

```diff
@@ -0,0 +1,234 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+"""ChangeStar implementations."""
+
+from typing import Any, Dict, List
+
+import torch
+from einops import rearrange
+from torch import Tensor
+from torch.nn.modules import (
+    BatchNorm2d,
+    Conv2d,
+    Identity,
+    Module,
+    ReLU,
+    Sequential,
+    UpsamplingBilinear2d,
+)
+
+from .farseg import FarSeg
+
+Identity.__module__ = "nn.Identity"
+
+
+class ChangeMixin(Module):
+    """This module enables any segmentation model to detect binary change.
+
+    'Change is Everywhere: Single-Temporal Supervised Object Change Detection',
+    Zheng et al. (2020)
+
+    If you use this model in your research, please cite the following paper:
+
+    * https://arxiv.org/abs/2108.07002
+    """
+
+    def __init__(
+        self,
+        in_channels: int = 128 * 2,
+        inner_channels: int = 16,
+        num_convs: int = 4,
+        scale_factor: float = 4.0,
+    ):
+        """Initializes a new ChangeMixin module.
+
+        Args:
+            in_channels: sum of channels of bitemporal feature maps
+                (default=256)
+            inner_channels: number of channels of inner feature maps
+                (default=16)
+            num_convs: number of convolution blocks
+                (default=4)
+            scale_factor: number of upsampling  factor
+                (default=4.0)
+        """
+        super(ChangeMixin, self).__init__()  # type: ignore[no-untyped-call]
```

Comment:
```suggestion
        super().__init__()  # type: ignore[no-untyped-call]
```
This is the Python 3 syntax for `super()`, Python 2 is long since dead.